upgrading/topics/keycloak/changes-19_0_0.adoc

```diff
@@ -258,4 +258,10 @@ spec:
     matchLabels:
       component: keycloak
 ```
-See also the https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Kubernetes Documentation].
\ No newline at end of file
+See also the https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Kubernetes Documentation].
+
+= Deployment changes in the new {project_operator}
+
+The new {project_operator} now uses `StatefulSet` instead of `Deployment` for Keycloak deployments. There's no automated
+migration in place given the Operator is a tech preview in this release. If you are using the new Operator with 18.0.z,
+please make sure to backup, delete and recreate your Keycloak CR after the upgrade to 19.
```

Comment:
@vmuzikar not a dealbreaker for merging it. Would make more sense to refer to 19.0.0, instead of just 19?

Reply:
Done, thank you for the review!